server/views/match/search.njk

```diff
@@ -58,7 +62,74 @@
 		</div>
 	</dl>
 	<hr/>
+
 	<h2 class="govuk-heading-m">{{bedSearchResults.resultsBedCount}} matching beds in {{bedSearchResults.resultsRoomCount}} rooms in {{bedSearchResults.resultsPremisesCount}} premises</h2>
+	<form action="{{formPath}}" method="post">
+		<div class="bed-search-inputs">
+			<input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+			<input type="hidden" name="crn" value="{{crn}}"/>
+			<input type="hidden" name="startDate" value="{{startDate}}"/>
+
+			{{
+				govukInput({
+				id: "durationDays",
+				name: "durationDays",
+				value: durationDays,
+				type: "number",
+				label: {
+				text: "Length of stay"
+				},
+				classes: "govuk-input--width-2",
+				suffix: {
+					text: "weeks"
+				}
+			})
+			}}
+			{{ govukDateInput({
+				id: "startDate",
+				namePrefix: "startDate",
+				fieldset: {
+				legend: {
+					text: "Expected arrival",
+					classes: "govuk-fieldset__legend--s"
+					}
+				},
+				items: dateFieldValues('startDate', errors),
+				errorMessage: errors.startDate
+			}) }}
+
+			{{ govukInput({
+				id: "postcodeDistrict",
+				name: "postcodeDistrict",
+				value: postcodeDistrict,
+				label: {
+				text: "Preferred location"
+				},
+				classes: "govuk-input--width-3"
+     		}) }}
```

Comment:
The alignment is a bit off here - is this the autoformatter just doing something janky, or is there an unclosed HTML tag somewhere?

Reply:
I think it's just the diff on Github, it looks ok locally for me